assets/delivery-options/js/myparcel.js

```diff
@@ -55,6 +59,16 @@ MyParcel = {
         MyParcel.bind();
     },
 
+    showPrices: function (prices) {
+        jQuery('#mypa-morning-delivery').html(MyParcel.getPriceHtml(prices.morning));
```

Comment:
hier kan je lopen door alle items. ipv ze allemaal apart te benoemen:
```
prices.each(function( selectName, price ) {
    jQuery('#mypa-' + selectName + '-delivery').html(MyParcel.getPriceHtml(price));
}
```
Ik weet niet of ik hier op de juiste manier loop, dat zou je even uit moeten zoeken.

Reply:
Sorry, dat ik dat niet duidelijk bij de vorige review had aangegeven.

Reply:
Done!